ui/apps/platform/cypress/integration/integrations/authenticationTokens.test.js

```diff
@@ -0,0 +1,126 @@
+import withAuth from '../../helpers/basicAuth';
+import { readFileFromDownloads } from '../../helpers/file';
+import {
+    getHelperElementByLabel,
+    getInputByLabel,
+    generateNameWithDate,
+    getSelectButtonByLabel,
+    getSelectOption,
+} from '../../helpers/formHelpers';
+
+import {
+    assertIntegrationsTable,
+    cleanupClusterRegistrationSecretsWithName,
+    clickCreateNewIntegrationInTable,
+    clickIntegrationSourceLinkInForm,
+    generateCreatedAuthProvidersIntegrationInForm,
+    revokeAuthProvidersIntegrationInTable,
+    visitIntegrationsTable,
+    visitIntegrationsDashboardFromLeftNav,
+} from './integrations.helpers';
+import { selectors } from './integrations.selectors';
+
+// Page address segments are the source of truth for integrationSource and integrationType.
+const integrationSource = 'authProviders';
+
+describe('Authentication Tokens', () => {
+    withAuth();
+
+    it('should create a new API Token and then view and delete', () => {
```

Comment:
This test is from the renamed file `apiTokens.test.js`. The only new code begins at the next `describe` block.